src/App.js

```diff
@@ -1,26 +1,26 @@
+import { useState } from 'react';
 import Card from './Card';
 import './index.css';
 
 const App = () => {
-  const books = [
-    {
-      title: 'Core Java Volume I – Fundamentals',
-      description: 'Core Java Volume I – Fundamentals is a Java reference book (Best book for Java)that offers a detailed explanation of various features of Core Java, including exception handling, interfaces, and lambda expressions. ',
-    },
-    {
-      title: 'Java: A Beginner’s Guide',
-      description: 'If you are a seasoned Java programmer looking to enhance your Java knowledge, don’t be averted by the title of the book, Java: A Beginner’s Guide.',
-    },
-    {
-      title: 'Head First Java',
-      description: 'The most important selling points of Head First Java is its simplicity and super-effective real-life analogies that pertain to the Java programming concepts.',
-    },
-  ];
+  
+  const [card, setCard] = useState({
+    title: 'Core Java Volume I – Fundamentals',
+    description: 'Core Java Volume I – Fundamentals is a Java reference book (Best book for Java)that offers a detailed explanation of various features of Core Java, including exception handling, interfaces, and lambda expressions. ',
+  });
+
+  const cardUpdateHandler = card => {
+    setCard({
+      ...card,
+      title: card.updatedTitle,
+      description: card.updatedDescription
+    });
+  }
```

Comment:
You're passing card with "updated" properties and use the same argument name as state const, so once used spread operator, you'll have a confusion, since spread operator is working on argument and passes updated properties as well.
![image](https://user-images.githubusercontent.com/52723192/159192641-e6c5e604-d1f9-4b68-a202-4b914ed8ed33.png)
As an option use the same properties names in "updated" as in original one.